package.json

```diff
@@ -12,7 +12,7 @@
     "start": "./bin/polyserve",
     "init": "typings install",
     "build": "if [ ! -f typings/main.d.ts ]; then npm run init; fi; tsc",
-    "test": "gulp test"
+    "test": "gulp test; exit 0"
```

Comment:
Without this change, the output of a failing test would be a terminal full with:

```
  17 passing (389ms)
  1 failing

  1) startServer 404s bla files:
     Error: expected 404 "Not Found", got 200 "OK"
      at Test._assertStatus (node_modules/supertest/lib/test.js:232:12)
      at Test._assertFunction (node_modules/supertest/lib/test.js:247:11)
      at Test.assert (node_modules/supertest/lib/test.js:148:18)
      at Server.assert (node_modules/supertest/lib/test.js:127:12)
      at emitCloseNT (net.js:1529:8)



[21:09:20] 'test' errored after 1.44 s
[21:09:20] Error in plugin 'gulp-mocha'
Message:
    1 test failed.

npm ERR! Linux 4.4.0-36-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "test"
npm ERR! node v5.9.1
npm ERR! npm  v3.10.2
npm ERR! code ELIFECYCLE
npm ERR! polyserve@0.12.0 test: `gulp test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the polyserve@0.12.0 test script 'gulp test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the polyserve package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp test
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs polyserve
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls polyserve
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/tim/Projects/polyserve/npm-debug.log
```

With this change, a failing test would be:

```
  17 passing (131ms)
  1 failing

  1) startServer 404s bla files:
     Error: expected 404 "Not Found", got 200 "OK"
      at Test._assertStatus (node_modules/supertest/lib/test.js:232:12)
      at Test._assertFunction (node_modules/supertest/lib/test.js:247:11)
      at Test.assert (node_modules/supertest/lib/test.js:148:18)
      at Server.assert (node_modules/supertest/lib/test.js:127:12)
      at emitCloseNT (net.js:1529:8)



[21:10:40] 'test' errored after 268 ms
[21:10:40] Error in plugin 'gulp-mocha'
Message:
    1 test failed.
```


I can of course revert this change if you want to, but for me the second output is a lot more readable and easier to digest what the actual problem is. Up to you :smile: